packages/teams-js/src/public/app.ts

```diff
@@ -354,6 +354,11 @@ export namespace app {
      * ID for the current visible app which is different for across cached sessions. Used for correlating telemetry data.
      */
     appLaunchId?: string;
+
+    /**
+     * App id that is used by Hosts to distinguish between different apps sideloaded or in store
+     */
+    appId: string;
```

Comment:
>string [](http://example.com/codeflow?start=11&length=6)

Can you use the `AppId` type for this? Strong types are better than strings. It should be easy to create this object as an `AppId` in `transformLegacyContextToAppContext` after the data comes in over the wire as a `string`.